classes/AllWeatherPortfolio.js

```diff
@@ -1,6 +1,10 @@
-import { PrimeSdk, EtherspotBundler } from "@etherspot/prime-sdk";
+import { smartWallet } from "thirdweb/wallets";
+import { arbitrum } from "thirdweb/chains";
```

Comment:
this PR is pretty hard to review since too many changes.

In short, I've sunsetted `Etherspot` and in favor of `thirdweb` for implementing `AA` (account abstraction)

Reply:
It seems like there have been many changes to the `thirdweb` library content.